Sources/PointFree/Episode/Show.swift

```diff
@@ -137,19 +145,34 @@ private let tocChapterView = View<(title: String, timestamp: Int, isEpisodeViewa
     ])
 }
 
-private let tocChapterLinkView = View<(title: String, timestamp: Int, active: Bool)> { title, timestamp, active -> Node in
+private let tocChapterLinkView = View<(title: String, timestamp: Int, active: Bool)> { title, timestamp, active -> [Node] in
   if active {
-    return a(
-      timestampLinkAttributes(timestamp) +
-        [`class`([Class.pf.colors.link.green, Class.type.textDecorationNone, Class.pf.type.body.regular])],
-      [text(title)]
-    )
+    return
+      [
+        div([`class`([Class.hide(.mobile)])], [
```

Comment:
we render two chapter links: one for mobile without anchors, one for desktop with anchors.

Reply:
Easy peasy.